src/ontology/nbo-edit.owl

```diff
@@ -1124,6 +1151,181 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/nbo#qualifier> "q
 
 
 
+############################
+#   Datatypes
+############################
+
+# Datatype: obo:NBO_0000010 (reproductive behavior)
+
+AnnotationAssertion(Annotation(obo:IAO_0000119 "Immelmann, K., & Beer, C. (1989). A dictionary of ethology. Harvard University Press.") Annotation(oboInOwl:hasDbXref "ISBN:0674205065") obo:IAO_0000115 obo:NBO_0000010 "A behavior process that tends to increase, “the propagation of new individuals”.")
```

Comment:
`“` looks like a bad char, better use `'`.

Reply:
I'm not sure where to change this: it isn't in the ROBOT template but seems to have been drawn in from the existing NBO entry about reproductive behavior, in which case the quotes were at one point accepted inside NBO. The same characters are also used in other places in this ROBOT template without being highlit, e.g. line 1274. Can the error be over-ridden or am I missing something?

Reply:
This appears to have been a template bug.